src/main/groovy/org/pih/warehouse/outbound/ShippingRequest.groovy

```diff
@@ -0,0 +1,27 @@
+package org.pih.warehouse.outbound
+
+import grails.validation.Validateable
+import org.pih.warehouse.shipping.ShipmentType
+
+class ShippingRequest implements Validateable {
+
+    Date expectedShippingDate
+
+    ShipmentType shipmentType
+
+    String trackingNumber
+
+    Date expectedDeliveryDate
+
+    static constraints = {
+        expectedShippingDate(validator: { value, obj ->
+            // Check if shipping date happens before the expected delivery date (if delivery date specified)
+            if (obj.expectedDeliveryDate && value.after(obj.expectedDeliveryDate + 1)) {
```

Comment:
why is the + 1 needed here? Is it adding a day?

Reply:
@EWaterman yeah, a similar addition is implemented in the Shipment domain.